lms/djangoapps/ora_staff_grader/ora_api.py

```diff
@@ -33,6 +34,36 @@ def get_submissions(request, usage_id):
     return json.loads(response.content)
 
 
+def get_assessments_given(request, usage_id, submission_uuid):
+    """
+    Get a list of assessments given from the ORA's 'list_assessments_given' XBlock.json_handler
+    """
+    handler_name = "list_assessments_given"
+    data = {"item_id": usage_id, "submission_uuid": submission_uuid}
+
+    response = call_xblock_json_handler(request, usage_id, handler_name, data)
+
+    if response.status_code != HTTPStatus.OK:
+        raise XBlockInternalError(context={"handler": handler_name})
+
+    return json.loads(response.content)
+
+
+def get_assessments_received(request, usage_id, submission_uuid):
+    """
+    Get a list of assessments received from the ORA's 'list_assessments_received' XBlock.json_handler
+    """
```

Comment:
I think we need a better explanation for each of these docstrings. Could we improve them? Without the right context, I wouldn't know what `assessments received` means.